anitya/lib/backends/npmjs.py

```diff
@@ -112,8 +117,11 @@ def check_feed(cls):
         except Exception:  # pragma: no cover
             raise AnityaPluginException('No JSON returned by %s' % url)
 
-        for item in data[:40]:
+        for item in data['results']:
+            if item.get('deleted'):
+                continue
+            item = item['doc']
```

Comment:
Minor nitpick, but this caught me a bit off-guard. Could you either change the variable name here to something like ``item_doc`` or just do ``item['doc']['name']``? At first I expected to find the 'name' key in the 'results' object.

Reply:
Fixed.